Makefile

```diff
@@ -65,13 +65,13 @@ build/init.sql: sql/privacy/init.sql
 update-js-deps:
 	find . -name node_modules -not -prune -or -name target -not -prune -or -name package.json -exec sh -c 'cd $$(dirname "$$0"); bun update --latest' {} \;
 
-CHECK_TARGETS=$(patsubst %/Skargo.toml,check-%,$(shell ls -1 */Skargo.toml))
-
-check-%:
-	cd $* && skargo check
-
 .PHONY: check
-check: $(CHECK_TARGETS)
+check:
+	find * -name Skargo.toml -exec sh -c 'bin/cd_sh $$(dirname {}) "skargo check"' \;
```

Comment:
This reminds me that we need a "workspace" notion in Skargo (like in Cargo and npm), which would allow a simple `skargo check` within the workspace root to check all packages.

Reply:
Yes! And hopefully it would know the interdependencies so that it could do each dir only once, and maybe even in parallel.

Reply:
@beauby another thing to consider regarding a workspace feature for skargo is that this PR's shell command doesn't actually work since what can and does happen is that we `cd` to a dir, run `skargo check`, skargo realizes that there is a dep so it tries to `skargo check` it, and in the case that there are errors while checking the dep, they are reported with locations relative to the dep's own Skargo.toml file, not relative to the directory from which the user ran `skargo check`. A workspace feature for skargo will need to teach the compiler to report errors relative to the workspace root.